src/pages/workspaces/workspace/workflows/DataStepContent.js

```diff
@@ -17,7 +17,7 @@ import validate from 'validate.js'
 const DataStepContent = ({
   entitySelectionModel, onDismiss, onSuccess,
   entityMetadata, rootEntityType,
-  workspace: { namespace, name, googleProject, attributes: { 'workspace-column-defaults': columnDefaults } }
+  workspace, workspace: { workspace: { namespace, name, googleProject, attributes: { 'workspace-column-defaults': columnDefaults } } }
```

Comment:
Instead of passing in the whole workspace, could you add the additional things you need from workspace to `workspace: { workspace: { namespace, name, googleProject, attributes: { 'workspace-column-defaults': columnDefaults } }`? It always seems icky to me when both the whole object and a selected subset of the same object are passed into something.

Reply:
Split this up in f37a36db147d20b47d0306872540eba609b9a5e6.